lib/testLauncher/SuitestLauncher.js

```diff
@@ -59,7 +59,17 @@ class SuitestLauncher {
 						},
 					};
 
-					log.startOnDevice(device.deviceId, this.restArgs.join(' '));
+					let ws = null;
```

Comment:
I would rename  `ws` to `writeStream` for readability.

Reply:
Agree, otherwise it can be confused with webSockets